app/controllers/professional_experiences_controller.rb

```diff
@@ -1,4 +1,8 @@
 class ProfessionalExperiencesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound do |_|
```

Comment:
if there's no use for the argument, you don't need to add here

` rescue_from ActiveRecord::RecordNotFound do` should be fine